emscripten/npm/src/verovio-logger.js

```diff
@@ -0,0 +1,15 @@
+export const logLevel = {
```

Comment:
With this [PR](https://github.com/rism-digital/verovio/pull/3132), the resutling calls will be:

C++:
```cpp
vrv::EnableLog(vrv::LOG_ERROR);
```
Python
```python
import verovio
verovio.enableLog(verovio.LOG_ERROR)
```

If we want to be inline with this and be able to do

JS
```js
verovio.enableLog(verovio.LOG_ERROR);
```
I assume we should do
```
export const LOG_OFF = 0;
export const LOG_ERROR = 1;
export const LOG_WARNING = 2;
export const LOG_INFO = 3;
export const LOG_DEBUG = 4;
```
And then (in `./src/index.js` and './prebundle-module.js`)
```
export default {
    LOG_OFF,
    LOG_ERROR,
    ...
}
```
It is not great to have to set each value explicitly, but I don't think we have another option.

Reply:
I would prefer the wrapping `logLevel` object. But of course I will refactor this accordingly.

What are the correct `int` values for these constants? I'm not a C++ developer so im not sure about this line: https://github.com/rism-digital/verovio/blob/57a7a4b778e275eace0d5d7a6eb9f6f33c11c679/include/vrv/toolkitdef.h#L32

```js
export const LOG_OFF = 0;
export const LOG_ERROR = 1;
export const LOG_WARNING = 2;
export const LOG_INFO = 3;
export const LOG_DEBUG = 4;
```

or what @ahankinson suggested in https://github.com/rism-digital/verovio/issues/3122#issuecomment-1315814132

```js
export const LOG_OFF = 0;
export const LOG_DEBUG = 1;
export const LOG_INFO = 2;
export const LOG_WARNING = 3;
export const LOG_ERROR = 4;
```

Reply:
And `EnableLogToBuffer` will still accept a `bool` and not the same `LogLevel` enum as `EnableLog`. Correct?

https://github.com/rism-digital/verovio/blob/57a7a4b778e275eace0d5d7a6eb9f6f33c11c679/src/vrv.cpp#L194-L197

Reply:
Okay, I pushed to change this (untested).